ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: vsphere-remote-worker
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  grace_period: 10m
+  commands: vsphere-remote-worker-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: ci-route-53
+    mount_path: /var/run/vault/vsphere
```

Comment:
seems this secret is not used in script?

Reply:
Good catch, I missed remove it.

Reply:
Fixed